### PR TITLE
Fix parsing for constructors and not objects

### PR DESCRIPTION
**Pull request by stek29**
Kinda like #81, but this time not for vector but for ipPort.
Tried to run tests to see if it breaks anything but looks like they're
not working lol.

A note about ipPort: it's used in help.configSimple, which is "backup DC" method.

Here's a test file:
```python
from telethon.extensions import BinaryReader
seqhex = """
c5 c3 97 d9 77 8d 9d 59 77 1a c5 59 04 00 00 00
15 c4 b5 1c 01 00 00 00 de a5 9a 95 bb 01 00 00
"""
seqb = bytes(int(h, 16) for h in seqhex.split() if h)
reader = BinaryReader(seqb)
conf = reader.tgread_object()
print(str(conf))
```

Before:
```
Traceback (most recent call last):
  File "testshit.py", line 8, in <module>
    conf = reader.tgread_object()
  File ".../telethon/extensions/binary_reader.py", line 135, in tgread_object
    return clazz.from_reader(self)
  File ".../telethon/tl/types/help.py", line 104, in from_reader
    _x = reader.tgread_object()
  File ".../telethon/extensions/binary_reader.py", line 133, in tgread_object
    raise TypeNotFoundError(constructor_id)
telethon.errors.common.TypeNotFoundError: (TypeNotFoundError(...), 'Could not find a matching Constructor ID for the TLObject that was supposed to be read with ID 0x959aa5de. Most likely, a TLObject was trying to be read when it should not be read.')
```

Now:
```
ConfigSimple(date=datetime.fromtimestamp(1503497591.0), expires=datetime.fromtimestamp(1506089591.0), dc_id=4, ip_port_list=[IpPort(ipv4=-1785027106, port=443)])
```

<sub>yes, it's current config tg is using. yes, it's "expired". yes, it's ok (at least they told me it is).</sub>

### CODE REVIEW

**Comment by stek29** [comment]
@Lonami please ~~put me on wall of shame~~ review :)

File: telethon_generator/tl_generator.py [hunk 1]
```diff
@@ -129,6 +129,9 @@ def _write_init_py(out_dir, depth, namespace_tlobjects, type_constructors):
                 builder.writeln(
                     'from {}.tl.tlobject import TLObject'.format('.' * depth)
                 )
+                builder.writeln(
+                        'from {}.tl import types'.format('.' * depth)
```
Comment (stek29) [inline telethon_generator/tl_generator.py]: will this slow down telethon? probably, idk.

File: telethon_generator/parser/tl_object.py [hunk 2]
```diff
@@ -98,12 +98,17 @@ def from_tl(tl, is_function):
...
         """Gets the class name following the Python style guidelines"""
+        return self.class_name_for(self.name, self.is_function)
 
+    @staticmethod
+    def class_name_for(typename, is_function=False):
+        """Gets the class name following the Python style guidelines"""
         # Courtesy of http://stackoverflow.com/a/31531797/4759433
-        result = re.sub(r'_([a-z])', lambda m: m.group(1).upper(), self.name)
+        result = re.sub(r'_([a-z])', lambda m: m.group(1).upper(),
+                typename)
```
Comment (Lonami) [inline telethon_generator/parser/tl_object.py]: I would do:
```python
        result = re.sub(
            r'_([a-z])', lambda m: m.group(1).upper(), typename
        )
```

But okay.

Comment (stek29) [reply telethon_generator/parser/tl_object.py]: @Lonami AAAAA Vim AUTO FOLDING

**Comment by Lonami** [comment]
> Tried to run tests to see if it breaks anything but looks like they're not working lol.

#323. Changes look fine, thanks!

**Pull request merged**